From 5556f5ff1c5b35d028a66ad50dc627af13e00c6d Mon Sep 17 00:00:00 2001
From: Oleksandr Andrushchenko <oleksandr_andrushchenko@epam.com>
Date: Tue, 19 Oct 2021 13:59:39 +0300
Subject: [PATCH] clk:shmobile: Hide clock for scif3

scif3 serial port is used by Xen, so do not let the kernel to
manage it.

Signed-off-by: Oleksandr Andrushchenko <oleksandr_andrushchenko@epam.com>
---
 drivers/clk/renesas/r8a779f0-cpg-mssr.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/clk/renesas/r8a779f0-cpg-mssr.c b/drivers/clk/renesas/r8a779f0-cpg-mssr.c
index 69e0a4ad65e8..4b5221c30b8c 100644
--- a/drivers/clk/renesas/r8a779f0-cpg-mssr.c
+++ b/drivers/clk/renesas/r8a779f0-cpg-mssr.c
@@ -143,7 +143,9 @@ static const struct mssr_mod_clk r8a779f0_mod_clks[] __initconst = {
 	DEF_MOD("rtdm3",	701,	R8A779F0_CLK_S0D2),
 	DEF_MOD("scif0",	702,	R8A779F0_CLK_S0D12_PER),
 	DEF_MOD("scif1",	703,	R8A779F0_CLK_S0D12_PER),
+#ifndef CONFIG_XEN
 	DEF_MOD("scif3",	704,	R8A779F0_CLK_S0D12_PER),
+#endif
 	DEF_MOD("scif4",	705,	R8A779F0_CLK_S0D12_PER),
 	DEF_MOD("sdhi0",	706,	R8A779F0_CLK_SD0),
 	DEF_MOD("sydm1",	709,	R8A779F0_CLK_S0D3),
-- 
2.33.0

